src/ast/ExecutionPathOptions.ts

```diff
@@ -17,15 +17,14 @@ export enum OptionTypes {
 	IGNORE_RETURN_AWAIT_YIELD,
 	NODES_CALLED_AT_PATH_WITH_OPTIONS,
 	REPLACED_VARIABLE_INITS,
-	RETRIEVED_VALUE_NODES,
 	RETURN_EXPRESSIONS_ACCESSED_AT_PATH,
 	RETURN_EXPRESSIONS_ASSIGNED_AT_PATH,
 	RETURN_EXPRESSIONS_CALLED_AT_PATH
 }
 
-export interface RESULT_KEY {}
-export const RESULT_KEY: RESULT_KEY = {};
-export type KeyTypes = OptionTypes | Entity | RESULT_KEY;
+interface RESULT_KEY {}
+const RESULT_KEY: RESULT_KEY = {};
+type KeyTypes = OptionTypes | Entity | RESULT_KEY;
 
 export class ExecutionPathOptions {
```

Comment:
Can `NEW_EXECUTION_PATH` be removed from this file with all these refactorings of it?

Reply:
Even though it is only used once in code, using the same variable would still provide some benefit. But I will move it to the proper location.